app/src/main/java/ch/epfl/skysync/components/forms/FlightForm.kt

```diff
@@ -592,6 +576,32 @@ fun RoleField(
       }
 }
 
+@Composable
+fun <T> TitledInputTextField(
```

Comment:
This could be extracted as a separated component, for ex: `components/TitledInputTextField.kt`

Reply:
I have done it for this and also for CustomDropDownMenu